example.java

```diff
@@ -141,6 +141,7 @@ private void integrationLayerPoint() {}
 	@Around("integrationLayerPoint()")
 	public Object integrationLayerLogAround(ProceedingJoinPoint joinPoint) {
 		boolean logMessage = false;
+		boolean logMessage1 = false;
```

Comment:
No need to declare. Boolean is by default false.

Reply:
I have fixed, please check